context/context.go

```diff
@@ -133,7 +134,12 @@ type fsContext struct {
 }
 
 func ConfigDir() string {
-	dir, _ := homedir.Expand("~/.config/gh")
+	configHome := "~/.config"
+	xdgConfigHome := os.Getenv("XDG_CONFIG_HOME")
+	if xdgConfigHome != "" {
+		configHome = xdgConfigHome
+	}
+	dir, _ := homedir.Expand(configHome + "/gh")
```

Comment:
Nit: if `configHome` was successfully read from XDG_CONFIG_HOME, you can skip `homedir.Expand()`, because there is no `~` character to expand.

Reply:
After update this function is removed, so I think this is outdated now.